src/DeepLinkModule/javasource/deeplink/actions/StartDeeplinkJava.java

```diff
@@ -138,7 +138,7 @@ public void processRequest(IMxRuntimeRequest request,
 				*	a new login page for the user instead of being logged in.
 				*/
 				if (request.getParameter(ARGUSER)!= null && request.getParameter(ARGPASS) != null) {
-                	if ( session == null || (session!=null && session.getUser().isAnonymous()) ) {
+                	if ( session == null || (session!=null && session.getUser(session.createContext()).isAnonymous()) ) {

```

Comment:
🎄 can't second `session!=null` check go?

```suggestion
if ( session == null || session.getUser(session.createContext()).isAnonymous() ) {
```

Reply:
No, either the session is null, or it's not null and the user is anonymous.

Reply:
New suggestion:
```suggestion
                	if ( session == null || session.getUser(session.createContext()).isAnonymous() ) {
```